web-console/5-components/order-from-catalog.md

```diff
@@ -114,19 +113,41 @@ Post 3.6, an additional Bind Step will be implemented.
 		- Mandatory fields should be indicated with an asterisk
 	- Optional "Show Advanced Options" link
 		- Available for some builder images
-	- Vertical scroll bar is displayed as needed, this scrolls the right side of the content
+	- Vertical scroll bar is displayed as needed
 - Buttons associated with this step
 	- Cancel, Back & Create buttons should be displayed
 	- Cancel should always be enabled
 	- Back should be enabled IF there is a previous Plan step
 	- Create should be disabled until all mandatory fields have been input
 
-### Step 4 - Results
+#### Form Layout
+- Labels are displayed before the input widget
+![template](img/configuration-step.png)
 
-#### Request has been submitted, and is in progress
-	** Mock needs to be updated **
+#### Form Layout - Mobile :iphone:
+- On mobile displays, labels are displayed above the input widgets
+
+	<img src="img/configuration-step-mobile.png" alt="alt text" width="200">
 
-![template](img/results-in-progress.png)
+#### Catalog item shows advanced options
+![template](img/config-step-hide-advanced.png)
+
+#### User views available projects in context of this flow
+![template](img/config-step-select-project.png)
+
+#### User creates a new project in context of this flow
+![template](img/config-step-create-project.png)
+
+### Bind Step (Optional)
+This step is only valid if the selected project is not new / has items that can be bound to.
```

Comment:
I think that as long as the service is bindable there should be a bind step. It may just be that the only bind option is to "create a secret" (as is the case in 3.7).